packages/forms/src/UIForm/fieldsets/CollapsibleFieldset/CollapsibleFieldset.component.js

```diff
@@ -29,7 +29,30 @@ function getDrillKey(key) {
 	}, []);
 }
 
+function isCollapsibleArray(element, schema) {
```

Comment:
`isCollapsibleArray` take 2 params, and only one is provided, is it normal?

Reply:
the second parameter is the main schema passed on the recursive calls within the same function

Reply:
Oh! Missed that part 😅